Weather/Views/WeatherViewController.swift

```diff
@@ -46,6 +50,9 @@ extension WeatherViewController: UICollectionViewDataSource, UICollectionViewDel
         if collectionView == view.timedForecastView {
             return timedForecastPresenter!.count
         }
+        if collectionView == view.weekForecastView {
+            return weekForecastPresenter!.count
```

Comment:
Avoid force unwrapping with nil coalescing operator (`??`)

Reply:
should I return 0 then? (`?? 0`)

Reply:
Yes, that is usually how it is done.